dev/com.ibm.ws.microprofile.health.2.0_fat/fat/src/com/ibm/ws/microprofile/health20/fat/MultipleHealthCheckTest.java

```diff
@@ -80,7 +96,23 @@ public void testSuccessReadinessCheck() throws Exception {
 
         JsonObject jsonResponse = getJSONPayload(conReady);
         JsonArray checks = (JsonArray) jsonResponse.get("checks");
-        assertEquals(1, checks.size());
+        assertEquals(2, checks.size());
+        assertEquals(true, checks.getJsonObject(0).getString("name").equals("successful-readiness-check") ||
+                           checks.getJsonObject(1).getString("name").equals("successful-readiness-check"));
+        assertEquals(jsonResponse.getString("status"), "UP");
+    }
+
+    @Test
+    public void testSuccessCDIProducerReadinessCheck() throws Exception {
+        log("testCDIProducerReadinessCheck", "Testing the /health/ready endpoint");
+        HttpURLConnection conReady = HttpUtils.getHttpConnectionWithAnyResponseCode(server1, READY_ENDPOINT);
+        assertEquals(SUCCESS_RESPONSE_CODE, conReady.getResponseCode());
+
+        JsonObject jsonResponse = getJSONPayload(conReady);
+        JsonArray checks = (JsonArray) jsonResponse.get("checks");
+        assertEquals(2, checks.size());
+        assertEquals(true, checks.getJsonObject(0).getString("name").equals("successful-cdi-producer-readiness-check") ||
```

Comment:
Similar as above comment, call a method and pass in the String array of the expected names.